### ISSUE TITLE
Control learning rate

### ISSUE THREAD

**Issue by mmgalushka** [mmgalushka/cvbench#34]
Currently we set learning rate and it is stay the same during all epochs. We need introduce option to decay the learning rate id the training metrics are flatten. Before doing any implementation we need to discuss this subject and identify the best strategy for implementation.
